README.md

```diff
@@ -6,18 +6,27 @@ This is simple demonstration project on how [Nats.io](https://nats.io/) can be i
 2. [NPM - Node Package Manager](https://www.npmjs.com/)
 3. [Nats.io](https://nats.io/)
 
+### Build, Setup and Run using script file
+
+```
+chmod +x RUN.sh
+./ RUN.sh
+```
+
+### Setup and Install without script
+
 ### Tips and Tricks
-If you are `Linux` user and well acquainted with [Docker](https://www.docker.com), you can simply set up the Nats Docker container using the following command
+If you are `Linux` user and well acquainted with [Docker](https://www.docker.com)
 ```
-docker run -p 4222:4222 -p 8222:8222 -p 6222:6222 --name nats-container -it nats:latest
+docker run -p 4222 -p 8222 -p 6222 --name nats-container -it nats 
 ```
 
-# Setting up the project
+### Setting up the project
```

Comment:
This is not required. Move back to the old heading style. This is because the docker setup is just quick trick but not the only way for nats to be installed